Dockerfile

```diff
@@ -2,17 +2,20 @@
 FROM python:2.7-slim
 
 # where the app will live in the container
-WORKDIR /usr/src/app
+WORKDIR /app
 
 # install any requirements to workdir
-COPY api/requirements.txt ./
+ADD requirements.txt /app
```

Comment:
will the requirements.txt live in the /api dir? maybe should move it there

Reply:
do this may be confusing since our app's folder structure is looks like

```
animoos
  - app/
  - api/
```

The dockerfile structure looks like this after this line of code
```
app/
  - requirements.txt
```

then on line 14 it moves the rest of the files (with exception of the (animoos') app folder into (container's) app folder